src/Tribe/Importer/Encoding.php

```diff
@@ -0,0 +1,338 @@
+<?php
+/*
+Copyright (c) 2008 Sebasti�n Grignoli
```

Comment:
How does the autoloader work from the vendor directory? Do I have to register it with it?

Adding it to that directory and renaming the class gives a class not found fatal